app/controllers/v0/form210779_controller.rb

```diff
@@ -1,26 +1,22 @@
 # frozen_string_literal: true
 
-# Temporary stub implementation for Form 21-0779 to enable parallel frontend development
-# This entire file will be replaced with the full implementation in Phase 1
-
 module V0
   class Form210779Controller < ApplicationController
     service_tag 'nursing-home-information'
     skip_before_action :authenticate
+    before_action :check_feature_enabled
+    before_action :load_user
```

Comment:
@annaswims :load_user needs to be called prior to :check_feature_enabled. These should be flipped.